Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -41,9 +41,12 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
 
     /// 12. Missing BA Token for App Switch
     case missingBAToken
-
+    
     /// 13. Missing PayPal Request
     case missingPayPalRequest
+    
+    /// 14. Missing EC Token for App Switch
+    case missingECToken
```

Comment:
It doesn't look like we return this error anywhere? Also if we removed the `missingBAToken` case we can remove it from our code and update the errors accordingly.

Reply:
ahh that's true, updated in 2415a612dc6679830cd04ee4a01ad1765932e291